packages/docs/src/stories/templates/drawer.stories.ts

```diff
@@ -29,55 +29,36 @@ export default {
 export const Default = {
   name: 'Tablet Navigation',
   render: () => html`
-    <style>
```

Comment:
I think removing the styles here broke the layout.

<img width="439" alt="Screenshot 2025-05-19 at 14 16 06" src="https://github.com/user-attachments/assets/e395337a-fbdf-4775-a22b-345d6eb1db8d" />